Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: dependencies
+dependencies: ## Install dependencies
+	pip install --user -r requirements.txt
```

Comment:
I run a virtualenv for python and `make dependencies` fails for me:
```
$ make dependencies
pip install --user -r requirements.txt
Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
....
```

This could be solved by optionally add the `--user` as follows:

```diff
$ git diff
diff --git a/Makefile b/Makefile
index 7d35d81..c66cc89 100644
--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@ help:
 
 .PHONY: dependencies
 dependencies: ## Install dependencies
-       pip install --user -r requirements.txt
+       pip install $(if ${VIRTUAL_ENV},,--user) -r requirements.txt
 
 .PHONY: build
 build: ## Builds the project

```

Reply:
reverted to the original command as it's Travis specific